src/main/java/com/conveyal/taui/persistence/FilePersistence.java

```diff
@@ -0,0 +1,31 @@
+package com.conveyal.taui.persistence;
+
+import com.amazonaws.services.s3.model.ObjectMetadata;
+import com.conveyal.taui.AnalysisServerConfig;
+import spark.Response;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+
+public abstract class FilePersistence {
```

Comment:
Javadoc here would help, especially in making it easier to differentiate this FIlePersistence from the class in R5.